codechain/subcommand/account_command.rs

```diff
@@ -163,3 +165,22 @@ fn read_password_and_confirm() -> Option<Password> {
         None
     }
 }
+
+fn read_network_id(chain: &str) -> Result<NetworkId, String> {
+    let mut contents = String::new();
+    let reader: &[u8] = match chain {
```

Comment:
Code duplication. Use `ChainType::scheme` method.

Reply:
Thank you. I changed it.